src/abi.rs

```diff
@@ -10,11 +10,6 @@ abigen!(
         function deleteIdentities(uint256[8] calldata deletionProof, uint32 batchSize, bytes calldata packedDeletionIndices, uint256 preRoot, uint256 postRoot) external
```

Comment:
Have we ever used this call on prod? If not I would rather we remove this variant and whatever env (dev/stage) it was used in we should nuke